### PR TITLE
Update /virt/install task to make it great again on RHEL8+

### PR DESCRIPTION
**Pull request by StykMartin**
Signed-off-by: Martin Styk <mastyk@redhat.com>

### CODE REVIEW

**Comment by StykMartin** [comment]
GuestONE - no method ks_meta
GuestTWO - method=http
Preserve method http defined by user in GuestTWO

python get_guest_info.py
OUTPUT - 4th param (virt-install --location)
<stripped> nfs://definitelynotredhat.com/BaseOS/aarch64/os/ <stripped>
<stripped> http://definitelynotredhat.com/BaseOS/aarch64/os/ <stripped>

python get_guest_info.py --location http
Output
<stripped> http://definitelynotredhat.com/BaseOS/aarch64/os/ <stripped>
<stripped> http://definitelynotredhat.com/BaseOS/aarch64/os/ <stripped>

python get_guest_info.py --location ftp
Output
<stripped> ftp://definitelynotredhat.com/BaseOS/aarch64/os/ <stripped>
<stripped> http://definitelynotredhat.com/BaseOS/aarch64/os/ <stripped>

File: virt/install/get_guest_info.py [hunk 1]
```diff
@@ -1,47 +1,84 @@
...
+
+import optparse
 import os
+import shlex
 import sys
-import xmlrpclib
-import xml.dom.minidom
 import time
+import xml.dom.minidom
+import xmlrpclib
```
Comment (p3ck) [inline virt/install/get_guest_info.py:14]: Should this import be in a try except block and fall back to xmlrpc.client to be compatible with python3?

Comment (p3ck) [reply virt/install/get_guest_info.py:14]: I see now that we explicitly call python2.  I suppose upgrading this to python3 would be an entirely new PR. :-)

**Review by p3ck — APPROVED** [review]
lgtm

File: virt/install/get_guest_info.py (hunk 1 above)
Comment (StykMartin) [reply virt/install/get_guest_info.py:14]: Yeah. Initially, I was thinking about python2/3. But this has to be done in another PR.
We can't even have shebang inside of files if we want to be fully compatible (thank you Fedora). I will create another PR to manage it. Pretty much shebangs remove, calling explicitly interpreter for distribution and use python-six for deps.

**Pull request merged**